sdcm/cluster.py

```diff
@@ -2150,6 +2150,11 @@ def scylla_setup(self, disks):
         result = self.remoter.run('/sbin/ip -o link show |grep ether |awk -F": " \'{print $2}\'', verbose=True)
         devname = result.stdout.strip()
         extra_setup_args = self.parent_cluster.params.get('append_scylla_setup_args')
+        result = self.remoter.run('sudo /usr/lib/scylla/scylla_setup --help')
+        if '--swap-directory' in result.stdout:
+            # swap setup is supported
+            extra_setup_args += ' --swap-directory / '
+
```

Comment:
May be it is better to implement as new method?

Reply:
if there were more places calling scylla_setup... , but I think there aren't

Reply:
> May be it is better to implement as new method?

@aleksbykov  I can send another PR to move this to a help function.

`def is_supported_option(cmd='scylla_setup', option='--swap-directory')`